core/src/sumeragi/network_topology.rs

```diff
@@ -92,19 +92,19 @@ impl Topology {
         for role in roles {
             match (role, self.is_non_empty(), self.is_consensus_required()) {
                 (Role::Leader, Some(topology), _) => {
-                    public_keys.insert(&topology.leader().public_key);
+                    public_keys.insert(topology.leader().public_key());
```

Comment:
how is this change relevant? in `core` we should prefer direct field access

Reply:
Using method lets us not to deref box manually